configsuite/config.py

```diff
@@ -202,42 +209,61 @@ def _build_transformed_layers(self):
         self._valid &= len(self._errors) == 0
         return layers
 
-    def _build_initial_merged_config(self, layers, schema):
+    def _build_initial_named_dict_merged_config(self, layers, schema):
+        rec = self._build_initial_merged_config
+        content_schema = schema[MK.Content]
+        config = {}
+        all_keys = (key for layer in layers for key in layer.keys())
+        for key in all_keys:
+            child_layers = tuple([layer[key] for layer in layers if key in layer])
+            if len(child_layers) == 0:
```

Comment:
Will this ever happen? Given that the `all_keys` are aggregated from each layers `keys`?